services/datamanager/builtin/capture_test.go

```diff
@@ -275,16 +285,15 @@ func passTime(ctx context.Context, mc *clk.Mock, interval time.Duration) chan st
 				close(done)
 				return
 			default:
+				time.Sleep(10 * time.Millisecond)
```

Comment:
I'm curious - how long is this test taking to run? Asking because this is passing time until the write buffer is full/flushes, and I'm not sure how many passes that takes. Just want to sanity check that it isn't unreasonably long

Reply:
The test only takes 0.54 seconds to run locally, so I think it should be fine!